Library of Alexandria side-story and new job types (#4770)
endless-sky/endless-sky: data/alexandria start.txt

mission "Alexandria Library Card: Offer"
	name "Alexandria Library Card"
	minor
	job
	description "Alexandria Library is looking for an Apprentice Seeker. Meet Daemos Titan at the Al-Qarawiyyin Library on Earth to help him seek the Chal Jub Paqmey. Payments will total 400,000 credits. Escorts will be provided through dangerous areas."
	to offer
		not "library card"
		"net worth" > 300000
		has "event: remembrance day"
		or
			not "chosen sides"
			has "main plot completed"
	source
		not planet Earth
		near Sol 7
	destination Earth
	on complete
		conversation
			`You arrive on Earth and take a series of shuttles to get to Fez, Morocco where you expect to meet Daemos Titan. As with all old cities, the outer regions are monstrous mega-buildings built in the last few centuries. As you move inward, you see skyscrapers of the 20th to 22nd centuries.`
			`	You turn a corner and see something you never thought you would see on the infamously-gray planet of Earth: a large, green, field of grass with a beautiful mosque. It looks like something out of Earth's ancient history, perfectly preserved in present day.`
			choice

> 				`	(Enter the library.)`

Author: Anyone who cares enough to research it will know that mosque holds the oldest library in the world.

> 				`	(Enter the library.)`

Reviewer: `<first> <last>` might know it, but the player is unlikely to.
Perhaps mention it, in the good ol' fashion of "As you know, the oldest library on Earth is housed in a mosque; that must be it."

> 				`	(Enter the library.)`

Author: Anyone wise enough to find the Al-Qarawiyyin Mosque in a forest of skyscrapers would be wise enough to guess that the Al-Qarawiyyin Library would be within the mosque of the same name.

> 			`On your way to the library, a scruffy-looking man approaches you. He has a rather large gun in his holster and a massive book strapped to his back. He starts staring intently at you, or possibly through you. His dark red sunglasses and porcupine beard make it difficult to read his face (or even see it). When the man abruptly stops walking you consider yelling for help, fearing you're about to be mugged or shot.`

Reviewer 2: The style is that *every* line after the first line in a conversation received an additional tab within the backticks, not every line after the first line after a choice. This error exists throughout the missions.
```suggestion
			`	On your way to the library, a scruffy-looking man approaches you. He has a rather large gun in his holster and a massive book strapped to his back. He starts staring intently at you, or possibly through you. His dark red sunglasses and porcupine beard make it difficult to read his face (or even see it). When the man abruptly stops walking you consider yelling for help, fearing you're about to be mugged or shot.`
```

Author: I have corrected the indentation throughout the file so that the first paragraph the player sees (and no other) will be indented in each conversation.

There is one place where there are two possible first paragraphs, depending on branch logic. In that case both of the possible first paragraphs are indented. The player will only see one of the two.

			`	Suddenly he seems startled as if from some terrible gastrointestinal distress. He dives to the nearest bench, pulls the gigantic book off of his back, and starts jotting notes. You're not sure he even noticed you. His mind had wandered to whatever metaphysical world scholars occupy when reality has no bearing on their lives.`
			choice
				`	(Interrupt him.)`
					goto interrupt
				`	(Watch for a few minutes and see what he does.)`
					goto watch

> 				`	(Slowly back away from the madman and fly back to the endless sky where you belong.)`

Author: The story is filled with game references and inside jokes. That will not change.

			`	His beard ruffles as he asks, "yes?"`
				goto question
			label question
			`	"Can you tell me where to find Daemos Titan?"`
			`	He looks baffled and asks, "What would you do if you found him?"`
			`	"I was sent to help him find a book."`
			`	"Right. Right. I asked the library for a second Seeker. Are you <last> <first>? Or was it <first> <last>? I'm so bad with names." He begins searching his belongings for another pencil while eyeing the scribbles in his book.`
			`	"Captain <last> will do. Are you Daemos Titan?"`
			`	"Yes, yes, yes. I'll get ready to leave. Meet me in the spaceport this evening."`

mission "Alexandria Library Card: First Volume"
	name "Alexandria Library Card"
	description "Take Daemos Titan to <destination> to search for the first of the five Chal Jub Paqmey."
	blocked `Daemos exclaims, "I can't sleep in a gun port! Come back when you have a spare bunk and space for 1 ton of cargo."`
	source Earth
	destination Shroud
	passengers 1
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Offer: done"
	on offer
		conversation

> 			`When you arrive, you see Daemos Titan eager to depart. He walks at a brisk pace towards your ship without even telling you his destination. Porters behind push a cart with an impressive number of books and at least a thousand pencils. You notice there is no modern equipment; not even a data pad.`

Author: No.

			`	"Let's go already," he says. "The five Chal Jub Paqmey are not getting any younger."`
			`	"Go where?"`
			`	"Yes, yes, yes. Details." He fumbles about his person and eventually finds a folded piece of paper in his beard. He shoves it at you so fast that his hand almost hits you. You're fairly certain it wasn't meant as a hostile gesture. This man simply hasn't talked to anyone other than himself for so long that he's forgotten basic social graces.`
			choice
				`	(Read the paper.)`
			`	The paper is of organic origin, not a modern synthetic, and is stamped with the Alexandria Library Seal using actual wax. The text is badly smudged, but you can make out some details.`
			`	Your job is to ferry Daemos around the galaxy looking for the five Chal Jub Paqmey. You'll receive a payment of 60,000 credits each time he finds one. When you return Daemos and all five Chal Jub Paqmey to Alexandria Station in the Markeb system, you'll receive another 100,000 credits and an Alexandria Library Card.`

> 			`	The library card will allow permanent access to the Alexandria Station and its library, no matter your legal status, nor the political situation. The library also offers Job Board jobs to card holders, and possibly a daily salary.`

Author: This is a paid internship, not a mission, as is explained in detail later. Paid internships always have benefits, payment, and job duties. (The stated job duties are always a small subset of the actual duties.)

			choice
				`	"What is a Chal Jub Paqmey?"`
					goto what
				`	"Let's go find the Chal Jub Paqmey."`
					goto accept
				`	"Go find another captain who will cater to your madness."`
					decline
			label what

> 			`	"They're a series of books that record an opera of a merchant-explorer named HoD Qav, wandering a world that seems much bigger than herself. Or was it himself? Anyway, nobody knows the ending since there were supposed to be eight books, but only five were made. All five were lost over the centuries and now I'm going to find them."`

Author: That is inconsistent with his character. He is stating a fact that he will find the books. Later he makes it clear that he's willing to die in the attempt. The player's refusal to help will not deter him.

			`	"Where are they?"`
			`	"If I knew, I wouldn't be searching for them! I just worked out where the first one is, but we'll have to scour space for the other four."`
			choice
				`	"Let's go find the Chal Jub Paqmey."`
					goto accept
				`	"Sorry, I don't have time for this."`
					decline
			label accept

> 			`	"Excellent. Our first destination is planet Shroud in the Alheka system. I suggest you take a route through the western Paradise Planets. We should be pretty safe there."`

Author: He has been an adventurer his whole life, and he is taking the player on a one-time guided tour of 1/3 of the galaxy. Also, in the opening scene, he is discovering where in the temple on Shroud the hidden entrance resides.

> 			`	"Excellent. Our first destination is planet Shroud in the Alheka system. I suggest you take a route through the western Paradise Planets. We should be pretty safe there."`

Reviewer: The suggestion to remove "planet" remains. Note that I remember seeing only "X in the Y system" and not "planet X in the Y system" in various missions.

				accept
	on visit
		dialog `It seems you left the insane librarian or some of his thousand pencils behind. Perhaps they're in one of your escorts?`
	on complete
		payment 60000

mission "Alexandria Library Card: Second Volume: Step 1"
	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."
	blocked `Daemos complains, "What do you think I am, an Anti-Missile Turret? I need a bunk and space for 1 ton of cargo."`
	source Shroud
	destination Tinker
	landing
	passengers 1
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: First Volume: done"
	on offer
		conversation

> 			`As Dameos guides you from the spaceport and you see a planet of anachronisms. In the high mountains, vast castles of medieval design house the pampered ultra-rich, living as if in a Victorian era. As you pass through the streets you notice all the comforts of the modern era concealed behind curtains: virtual reality booths, food-synthesizing devices, massive panel screens for tracking stocks, and the like. When you leave the wealthy areas, you see the poor living in cramped conditions no better than medieval serfs, using modern technology only in their workplaces.`

Author: The original text sends the message of the story better than the overly-wordy one the reviewer suggested.

			choice
				`	(Continue.)`

> 			`Daemos isn't interested in any of that though; he rushes you on before you can fully take in the environment. You race through the cloud cities of the rich, beyond the factory towns, past the countryside, deep in a marsh, into a hidden temple, beneath a fake support column, and down to a subterranean room. In the center of the room, an impossibly old monk is meditating with his eyes closed. He hovers a half meter above the ground humming in an almost guttural tone. Within his hands is a book.`

Author: The original is correct, and any changes in wording are not acceptable.

			choice
				`	(Approach the monk.)`
			`When Daemos approaches, the monk's eyes open, and he utters, "For the Seeker I do seek." He hands Dameos the book and then vanishes.`
			`	Daemos doesn't surprised at all; he calmly puts the book in his satchel and handed you 60,000 credits.`
			`	"We have the Paq wa'DIch. Next we seek the Paq cha'DIch."`
			`	This seemed too easy for a 60,000 credit mission. You suspect the next four books will be harder to get.`
			`	Daemos wanders off, and you see him reading the book. He probably won't listen to you for a few hours.`
			choice
				`	(Wait a few hours.)`

> 			`You find Daemos later on, sitting in a tree the book on his lap. It's hard to see beneath his beard, but you think he's smiling.`

Author: No.

> 			`	You find Daemos later on, sitting in a tree the book on his lap. It's hard to see beneath his beard, but you think he's smiling.`

Author: Your suggested text change removes the information that significant amounts of time has passed.

			`	"Are you ready to go, Captain <last> <first>?"`
			choice
				`	"I have some questions first."`
					goto questions
				`	"Let's go."`
					goto leave
			label questions
			`	"What do you want to know?"`
			choice
				`	"What was in the book?"`
					goto contents
				`	"How will you find the next book?"`
					goto seek
				`	"Who was that monk, and where did he go?"`
					goto monk
				`	"No more questions. Let's go."`
					goto leave
				`	"I'm done with your quest. Enjoy the book."`
					decline
			label contents

> 			`"The book begins a story about a merchant, HoD Qav of Clan Botolphstone, destined to be the greatest warrior of the era. Through hard work, HoD Qav left the lands of Botolphstone, surviving on dangerous roads through wit alone. HoD Qav became an explorer, experiencing the vast continent on which their clan resides."`

Author: Traversing suggests the explorer plowed through the continent ignoring everything. To HoD Qav, it was an experience.

Did you read the story, or just check for grammar errors?

			`	"Why do you say 'their' clan instead of his or her?"`
			`	"The opera doesn't specify the gender of the merchant. The way it's written, the merchant could be male or female... or both. Or neither, I suppose."`
			`	"What happens next in the story?"`
			`	Daemos looks at you as if you were a child. "To know what happens next, we'll have to find the second book: the Paq cha'DIch."`
				goto questions
			label seek

> 			`	"My research suggests that the next book deals with war, unfairness, and greed. It is filled with mystery and amorality. That is also the very definition of the Syndicate. Their Director of Manufacturing is famous for his vast library."`

Author: That is not consistent with his character. Daemos's moral code is not the same as yours.

				goto questions
			label monk
			`	"The monk? What monk? Haven't you ever seen a hologram before?"`
				goto questions
			label leave

> 			`	"Excellent. We're going to Tinker in the Diphda system. It's at the edge of Syndicate space. Some stars around here are haunted by pirates. Take a route through the Paradise Planets and then Sol."`

Author: The word, "here," refers to the general area, not Diphda specifically. Saying, "to stay safe," is inconsistent with Daemos's character.

				accept
	on visit
		dialog `You've forgotten your senile friend or his pencils. Perhaps they're in one of your escorts?`

mission "Alexandria Library Card: Second Volume: Step 2"
	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."

> 	blocked `Daemos complains, "I can't sleep in cooling ducts. I need a bunk and space for 1 ton of cargo."`

Author: Fixed in latest commit.

	source Tinker
	destination Amazon
	landing
	passengers 1
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Second Volume: Step 1: done"
	on offer
		conversation

> 			`You are led to a vast skyscraper overlooking a mountain range. From the glass-walled elevator, you take in the breathtaking view. The elevator stops on the top floor and you see a well-dressed man surrounded by bookshelves. He sitting on a gold-lined leather chair, pulled up to a marble desk. The bookshelves also appear to be hand-carved marble. Whoever this is, he is exceptionally rich.`

Author: The "also" underlines the character's surprirse at the expense of hand-carving bookshelves, not just the desk.

			choice

> 				`	(Interrupt him.)`

Author: The player is along for the ride, so conversation options with characters won't happen without good reason. The "choice" block gives them a break in the conversation so there is an abrupt pause between the surprise at the contents of the room, and the more mundane conversation that follows. That ensures they notice the expense of the surroundings rather than skipping to the next paragraph.

			`The man turns to you and says, "It is always an honor to meet Alexandria Seekers. How can I help you?"`
			`	Daemos responds, "I seek the Paq cha'DIch, second volume of the Chal Jub Paqmey. I've heard it's your kind of literature. Do you have it?"`
			`	"I don't have it now, but I can get it for you. Do you have the book, 'A Thousand Poisons of Amazon and How to Conceal Their Taste?' I've been looking for a signed first edition."`

> 			`	"Ekhh. A trade. Okay, I'll get you a copy."`

Author: In english transliterations, "kh" is a hard "h" sound, which does not exist in english. It does exist in Klingon, however.

"Ehhh" is an expression of nervousness and is not consistent with Daemos's character, nor would it make sense given the situation.

"Ekhh" is an expression of annoyance or disgust.

			`	Daemos whispers to you, "We have to fly to planet Amazon in the Schedar system next. I have the book, but we need to get it signed. I have someone to contact... I'll see you at your ship."`
			choice
				`	(Continue.)`

> 			`	Daemos meets you at your ship and says, "Our destination is in a dangerous area of Syndicate space. I've arranged some escorts so you don't get me killed."`

Author: The break is necessary because the location has changed. It gives a sense of time passing.

> 			`	With a disgruntled expression, he asks, "Are you going to pester me with questions again?"`

Author: Any player that does not care enough about the story to ask questions won't care much about that. Any player that does ask questions deserves attention to details in the interactive story-telling.

				goto "actual questions"
			label questions
			`	"Are you going to pester me with questions again?"`
			label "actual questions"
			choice
				`	"What will we do on Amazon?"`
					goto what
				`	"Why do we need escorts?"`
					goto why
				`	"Sounds good. Let's go."`
					goto go
				`	"I don't have time to help you any more."`
					decline
			label what
			`	"I have a first edition of the book that the executive wanted, but we need to get it signed. It's a popular book among the Syndicate, so I always keep a copy or two when I visit their aristocracy. I'm a personal friend of the author, and I know the location of his remote hermitage."`
				goto questions
			label why

> 			`	Daemos responds, "That area is crawling with pirates and I doubt your ship will survive alone. The Chal Jub Paqmey are critical, so the Library is willing to expend significant resources to obtain it."`

Author: This was wrong in only two places, and one was intentional. 

The incorrect question from the player is because the player does not know whether the name is plural or not. It is best to assume it is singular, as the object the quest seeks, rather than a group of such objects. Daemos now rudely corrects the player.

> 			`	"Why am I your courier if you fear I'll get you killed?"`

Author: A `choice` for every question would get tedious.

			`	"Courier? You're not a courier. I don't understand."`
			`	"You're paying me to ferry you around the galaxy looking for books."`
			`	"No. You don't understand. You're my apprentice. Alexandria Library doesn't ask just anyone to assist a Grandmaster Seeker. They think you have potential to be a Seeker, despite all your faults. The payments are just stipends to keep you fed during your apprenticeship."`
			label "seeker questions"
			choice
				`	"What is a Seeker?"`
					goto seeker
				`	"Why me?"`
					goto "why me"
				`	"What faults?"`
					goto faults
				`	(Change the subject.)`
					goto questions
				`	"Sounds good. Let's go."`
					goto go
				`	"I don't have time for this. I'm out."`
					decline
			label seeker

> 			`	"An Alexandria Seeker is an adventurer who scours the galaxy for works of literature or art and brings them back to the library for archival. It's a dangerous job, but it is worth the risk to ensure knowledge is preserved.`

Author: I'd rather not spam the log, but I'll add one or two for the mission as a whole.

> 			`	"An Alexandria Seeker is an adventurer who scours the galaxy for works of literature or art and brings them back to the library for archival. It's a dangerous job, but it is worth the risk to ensure knowledge is preserved.`

Author: I'm still pondering log entries. I'll get to them in 6-10 hours. All I can do at my current location is edit text files. I want to see the log entries in the game before I commit them.

				goto "seeker questions"
			label go
			`	"Excellent! Now go to the spaceport to meet our escorts."`
				accept
	on decline
		fail "Alexandria Library Card: Second Volume: Step 2 Escort"
		fail "Alexandria Library Card: Second Volume: Step 2 Jump Instructions"
	on fail
		fail "Alexandria Library Card: Second Volume: Step 2 Escort"
		fail "Alexandria Library Card: Second Volume: Step 2 Jump Instructions"
	on visit
		dialog `It seems you left the insane librarian or some of his thousand pencils behind. Perhaps they're in one of your escorts?`
	npc
		government "Pirate"
		personality vindictive plunders staying
		system Schedar
		fleet
			names "pirate"
			variant
				"Fury"
				"Wasp"
				"Quicksilver"
	

mission "Alexandria Library Card: Second Volume: Step 2 Jump Instructions"
	source Tinker
	destination Amazon
	invisible
	to offer
		has "Alexandria Library Card: Second Volume: Step 1: done"
		has "Alexandria Library Card: Second Volume: Step 2 Escort: accepted"
		not "Alexandria Library Card: Second Volume: Step 2: declined"
		not "Alexandria Library Card: Second Volume: Step 2: failed"
	on enter

> 		dialog `You have acquired some escorts! Make sure you jump with them. Hold the JUMP button (default is "j") until all of your escorts' icons turn white. Yellow means they're not ready to jump. White means they're ready to jump. Blue means they're either in a different system, they're disabled, or they're docked fighters. Red means they are out of fuel or unable to jump. You can refuel them by landing or by boarding them with shift-BOARD (default is shift-"B")`

Author: The purpose of the text is to make the information immediately available, not in some distant help screen. This is a tutorial story.

	outfits
		"nGVF-BB Fuel Cell"
		"Heavy Laser"
		"LP036a Battery Pack"
		"D14-RN Shield Generator"
		"A120 Atomic Thruster"
		"Chipmunk Plasma Steering"
		"Hyperdrive"
		"Supercapacitor"

mission "Alexandria Library Card: Second Volume: Step 3"
	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."
	blocked `Daemos reminds you, "I won't fit in a torpedo. I need a bunk and space for 1 ton of cargo."`
	source Amazon
	destination "Deadman's Cove"
	passengers 1
	clearance "Daemos sends a message to his friend on Deadman's Cove. You receive coded landing coordinates."
	landing
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Second Volume: Step 2: done"
	on enter Almach
		dialog `Press "L" to land and then "shift-T" to talk to the planet.`
	on decline
		fail "Alexandria Library Card: Second Volume: Step 3 Escort"
	on fail
		fail "Alexandria Library Card: Second Volume: Step 3 Escort"
	on visit
		dialog `It seems you left the insane librarian or some of his thousand pencils behind. Perhaps they're in one of your escorts?`
	on offer
		conversation

> 			`Instead of landing on the main Amazon spaceport, Daemos has you land deep inside one of the densest forests on the planet. You climb through some aggressive vegetation, up a mountain, to a hut. Daemos looks at you and says, "This is the place."`

Author: The grammar errors were intentional. A rapid string of words like that gives the reader a sense that the characters are hurrying.

			label "slayer questions"
			`"Do you want to know anything before we go?"`
			choice
				`	"What kind of name is 'Crimson Slayer?'"`
					goto name
				`	"How will a fleet of Sparrows keep us alive?"`
					goto sparrows
				`	"Will she escort us back?"`
					goto back
				`	(Change the subject.)`
					goto questions
				`	"Sounds good. Let's go."`
					goto go
				`	"I don't want to die. Find another captain."`
					decline
			label name
			`"That's exactly the kind of question you should not ask her if you want to stay alive."`
				goto "slayer questions"
			label sparrows
			`"With missiles and upgraded thrusters, a fleet of Sparrows can use highly-effective swarm tactics. Besides, with that many escorts, the bounty hunters will have enough targets that they'll have trouble hitting us."`
				goto "slayer questions"
			label back
			`"Ah.... no. Crimson Slayer has other places to be"`
			`	"So you're saying we won't need help getting back?"`

> 			`	"Uh... well... Departing a planet is far more dangerous than landing. Pirates will wait just outside of a planet and destroy you before you can even react. A fast ship with good shields can dodge if it curves away from the planet as soon as it leaves orbit. A ship with decent shields, good steering, and a Scram Drive can jump immediately. Not your ship though. Certainly not there.`

Author: This is a multi-line quote.

	to offer
		has "Alexandria Library Card: Second Volume: Step 3: done"
	on visit
		dialog `You've forgotten your senile friend or his pencils. Perhaps they're in one of your escorts?`
	on decline
		fail "Alexandria Library Card: Second Volume: Step 4 Escort"
	on fail
		fail "Alexandria Library Card: Second Volume: Step 4 Escort"
	on offer
		conversation
			`Daemos has you land on a remote island near the southern pole. The buildings here are well-concealed, durable structures. You enter one and go down an elevator for at least two minutes into some kind of hidden shipyard. A large number of men in masks seem to be modifying a Bounder, adding space for a gun.`
			`	One of the men comes towards Daemos, growling. It's clear that he wishes he could kill you both.`
			`	"I'm here for a signature," says Daemos. He seems completely unfazed by the man.`
			`	You're brought to a room with a much taller man in a mask. You've never seen someone this large or this muscular before.`
			choice
				`	(Enter the room.)`
			`	The large man speaks, "We need a legitimate reason to enter Syndicate space. Can you arrange that?"`
			`	"Certainly," says Daemos.`
			`	"Then I can arrange a signature."`

> 			`	Daemos turns to you and says, "Meet me at the spaceport when you are ready to return to <planet>. Don't take long, and don't fall asleep in the hotels."`

Author: Every pirate planet has hotels, and comes with the occasional warning, "don't fall asleep in the hotels." Daemos is just repeating it. This is one of many inside jokes that mission creators will understand. Another is the main character's name, which translates to "Captain Last."

mission "Alexandria Library Card: Third Volume: Step 2"
	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to see an xenolinguist."
	blocked `Daemos complains, "I can't sleep in your reactor core. I need a bunk and space for 1 ton of cargo."`
	source Rand
	landing
	destination Grakhord
	passengers 1
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Third Volume: Step 1: done"
	on offer
		conversation
			`After landing on Rand, Daemos takes you to a circle of stones on a mesa near one of the largest mining towns. There are odd markings on the stones which Daemos insists must be words, even though he's never seen writing like that before.`
			`	Two hours and thirty books later, Daemos tells you, "It's no good. I'm going to have to talk to some xenolinguistics experts. We need to go to Grakhord in the Sadalmelik system. There's a Navy group that patrols along here regularly. I'm sure they wouldn't mind going two systems past the end of Human space."`
			`	Your interest piques when you hear, "the end of Human space."`

> 			`	"Oh, no, Captain <last> <first>. I'm not spoiling that surprise.`

Author: Daemos is senile, and mixes up names.

			`	Sophia hands the map to Daemos. You look over his shoulder.`
			choice
				`	(Look at the map.)`
			`	It's instantly clear what you are looking at. This is an old astronavigational system that uses constellations as seen from galactic center. It's reassuring that your pilot training taught you something Quarg and Human linguists couldn't figure out.`
			choice
				`	(Explain the symbols to them.)`
					goto explain
				`	(Let them suffer for a while.)`
			`	"Yes, these symbols do look almost like constellations," Daemos says. "Without knowing the era of the galaxy, we can't know where they should be."`
			`	Sophia adds, "I had some friends run this through stellar drift simulations, and predict what the constellations would look like everywhere in this region of space. Nothing came up."`
			`	"Well, Captain <last> <first>, if we can't figure this out, I guess we'll have to give up. I'll get the library to research this. I'm sure we'll come up with an answer in a year or two."`
			choice
				`	(Explain the symbols to them.)`
				`	"Oh well. I'll get back to my life as a merchant."`
					decline
			label explain

> 			`	You say, "It's an old navigational system taught to all pilots. The constellations are seen from the center of the galaxy. The crosses and ovals show the planet or moon."`

Author: I'm going with, "The constellations are drawn as they look from the center of the galaxy." I'll fix that in a commit with a batch of changes soon.

> 			`	You say, "It's an old navigational system taught to all pilots. The constellations are seen from the center of the galaxy. The crosses and ovals show the planet or moon."`

Author: Fixed in latest commit.

> 			`	"Of course!" says Sophia. "The constellations would change slower there due the distance to the stars. It's obvious which ones we go to."`

Author: That would be inconsistent with her character. She feels the need to seem knowledgeable after a mundane pilot outsmarts her.

			`	"That doesn't tell us where the keys are on the planets though," warns Daemos.`
			`	Sophia smiles, "On that matter, I can help you. My Quarg collaborators say the keys should appear on the Navy's gamma ray sensors as glitches, filtered out by quality control code. They already have the locations in their sensor logs' raw data stream. You just have to tell them the frequencies to look for."`
			`	She hands you a data pad and says, "Take this to the Navy research center on New Austria. They'll know what to do."`
			choice

> 				`	(Continue.)`

Author: It is also used to avoid spamming the character with a wall of text. Ten paragraphs on one page is a good way for your text to be ignored.

	on visit
		dialog `You've forgotten your senile friend or his pencils. Perhaps they're in one of your escorts?`
	on decline
		fail "ZZ Alexandria Library Card: Third Volume: Step 3 Escort"
	on fail
		fail "ZZ Alexandria Library Card: Third Volume: Step 3 Escort"
	npc
		government "Pirate"
		personality vindictive plunders staying
		system Menkent
		fleet
			names "pirate"
			variant
				"Marauder Leviathan (Engines)" 1
				"Aerie" 2
				"Dagger" 4

mission "ZZ Alexandria Library Card: Third Volume: Step 3 Escort"
	name `Escort Fleet: Tarazed Shipyards`
	description `Tarazed Shipyards corporate ships will escort you to <destination>. After <date>, they'll have to return to their patrol.`
	source Grakhord
	destination "New Austria"
	deadline 19
	to offer
		has "Alexandria Library Card: Third Volume: Step 2: done"
		not "ZZ Alexandria Library Card: Third Volume: Step 3: declined"
		not "ZZ Alexandria Library Card: Third Volume: Step 3: failed"
	on offer
		conversation
			`A saleswoman in a Tarazed Shipyards uniform approaches you. "We have business near the Navy research center so we'll escort you to <destination>. If you don't arrive by <date>, we'll have to part ways.`

> 			`	"By the way, I noticed you may be in need of a new ship. You'll pass by the Tarazed system when you exit Quarg space. Our corporate capital there, on Wayfarer, has the most well-stocked shipyard and outfitters in Human space. Stop by if you have a chance."`

Author: The world ,"Human," refers to a species and is a proper noun. Both "shipyard" and "outfitter" should be plural in that situation. I will fix this in a later commit.

> 			`	"By the way, I noticed you may be in need of a new ship. You'll pass by the Tarazed system when you exit Quarg space. Our corporate capital there, on Wayfarer, has the most well-stocked shipyard and outfitters in Human space. Stop by if you have a chance."`

Author: Actually, it's broken either way. I'll play through to that point in the story and see how I can reword it.

> 			`	"By the way, I noticed you may be in need of a new ship. You'll pass by the Tarazed system when you exit Quarg space. Our corporate capital there, on Wayfarer, has the most well-stocked shipyard and outfitters in Human space. Stop by if you have a chance."`

Author: I found a better sales pitch.

				accept
	npc
		government "Neutral"
		personality heroic escort disables
		fleet
			names "civilian"
			variant
				"Falcon"
				"Osprey" 3
				"Blackbird" 5

mission "Alexandria Library Card: Key"
	name "Alexandria Library Card"
	description "Gather key segments from <stopovers> and bring all five to <destination>."
	blocked `Daemos complains, "I won't fit in your missile launcher. I need a bunk and space for 1 ton of cargo."`
	source "New Austria"
	destination Rand
	landing
	stopover "New Boston"
	stopover "New Holland"
	stopover "Deep"
	stopover "Trinket"
	stopover "New Iceland"
	passengers 1
	cargo "old books and a thousand pencils" 1
	to offer
		has "ZZ Alexandria Library Card: Third Volume: Step 3: done"
	on offer
		conversation

> 			`You land and walk towards the Navy research center. A collection of concerned-looking, and well-armed, guards approach you. Seeing Daemos, they wave you on, and you meet with a sensor data analyst.`

Author: The extra commas were intentional, to emphasize those words. Long dashes and italics are not possible, so extra commas are the best I can do.

			choice
				`	(Talk to the analyst.)`

> 			`	He glances at your data pad and immediately knows what happened. "Coherent dichromatic gamma ray emissions from a planet are quite rare. We assumed they were glitches in the detector. Sometimes we'll get a frequency-shifted ghost image if the luminosity in one sensor cell is too high. We have to discard 2.3 microseconds of data every time that happens. You see, you get a double emission because it excites neu... uh... I shouldn't tell you that..."`

Author: Some things are obvious to a scientist with fifty years of experience in atmospheric remote sensing, which may not be obvious to you. This is especially true of rare, but interesting, phenomena that have caused major problems in quality control development. Some examples from present day are pyroconvection, algae blooms, or deliberate errors in land usage databases from cold-war-era decisions to hide military bases. Dealing with such issues can take hundreds of hours of work manipulating equations, developing code, and testing it through expensive measurement campaigns. Even then, some issues are impossible to work around.

A dichromataic gamma ray emission from a planet would be downright strange, and it is clear that it causes major quality control issues and losses of military-critical data. A dozen people in the room would recognize it in seconds. It was probably the subject of several research papers.

I'm guessing you haven't been involved in atmospheric remote sensing quality control algorithm development.

			`	"I've forgotten it already," says Daemos.`
			`	"The search algorithm has combed the data on the planets you listed for a 2,710,000 second interval. Assuming these keys don't move often, we can tell you where they are to a precision of 3.0994 millimeters. It would be finer except for the feedback we get from the... right... I shouldn't mention that either..."`
			`	The analyst adds geographical locations to your data pad and scurries away, hoping to avoid giving out any more Navy secrets.`
			choice
				`	(Continue.)`

> 			`Daemos seems eager to move on. "Well, Captain <first>, we know where to go. Are you ready? Or will you pester me with questions again?"`

Author: Daemos is senile and mixes up names.  Seriously, did you read the story at all?

				goto choice
			label top
			`	"Are you ready? Or will you pester me with questions again?"`
			label choice
			choice
				`	"Will we have escorts?"`
					goto escorts

> 				`	"What did read in the book we just recovered?"`

Author: It was recovered in the previous mission. I'll clarify that in the text.

> 				`	"What did read in the book we just recovered?"`

Author: Fixed in latest commit.

	source "New Austria"
	destination Rand
	deadline 30
	to offer
		has "ZZ Alexandria Library Card: Third Volume: Step 3: done"
		not "Alexandria Library Card: Third Volume: Key: declined"
		not "Alexandria Library Card: Third Volume: Key: failed"
	to complete
		has "Alexandria Library Card: Key: done"
	on offer
		conversation
			`Some men in green uniforms approach. One of them addresses you, "Captain <last>. I'm Commodore Garret of the New Boston Militia. The Alexandria Library is paying for us to escort you until <date>."`
				accept
	npc
		government "Militia"
		personality heroic escort frugal disables
		fleet
			names "militia"
			variant
				"Osprey" 1
				"Bastion" 2
				"Fury" 3
				"Hawk" 4
		

mission "Alexandria Library Card: Return"
	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to pick up the last book of the Chal Jub Paqmey."
	blocked `Daemos complains, "I'm not generator fuel. I need a bunk and room for 1 ton of cargo."`
	source Rand
	destination Valhalla
	passengers 1
	landing
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Key: done"
	on offer
		conversation

> 			`Daemos assembles all five segments into one sphere and places it in the ground. A massive red light comes down from the sky and the sphere is replaced by a book.`

Author: No. That is a baseless assertion.

			`	"Oddly, instructins in the Chal Jub Paqmey say the third and fourth books can be reversed when the opera is performed. HoD Qav can explore the world and gather power, and then return to their continent to bring peace. Or, the merchant can bring peace to their continent and then explore the world."`
				goto questions
			label accept
			`"Finally."`
				accept
	on visit
		dialog `You've forgotten your senile friend or his pencils. Perhaps they're in one of your escorts?`

mission "Alexandria Library Card: Rescue"
	name "Alexandria Library Card"
	waypoint Mora
	description `Repair the "Gonza Book Cruise" in the Mora system and take it to <destination> before <date>.`
	blocked `Daemos informs you, "I don't fit in a glove compartment. I need a bunk and space for 1 ton of cargo."`
	source Valhalla
	destination Ingot
	passengers 1
	landing
	deadline 11
	cargo "old books and a thousand pencils" 1
	on offer
		conversation
			`A Deep Security officer runs towards you and Daemos in a hurry yelling, "It's been disabled! We have to leave!"`

> 			`	When the officer reaches you, he explains the Alexandria Seeker who was supposed to meet you is in grave trouble. The Seeker's ship is disabled in an uninhabited system called Mora.' The Seeker, Gonza Doorin, refuses to let anyone onto his ship except another Seeker. There is a pirate fleet in the system, attacking anyone who gets near the ship."`

Author: No.

			`	Daemos interjects, "Can Deep Security escort us?"`

> 			`	"Of course," the officer responds. "We must leave at once.`

Author: This is a multi-line quote. Also, verbosity would be incorrect given the character's obvious rush.

			choice
				`	"Agreed."`
					accept
				`	"Rescue your own ship. I have better things to do than save self-important librarians."`
					decline
	to offer
		has "Alexandria Library Card: Return: done"
	on complete
		payment 60000
	on visit
		dialog `You need to bring Daemos, his cargo, and the Gonza Book Cruise here to complete this mission.`
	on decline
		fail "Alexandria Library Card: Rescue: Escort"
	on fail
		fail "Alexandria Library Card: Rescue: Escort"
	npc accompany save
		personality escort derelict timid target
		government "Librarian"
		ship "Star Queen" "Gonza Book Cruise"
		system Mora
	npc
		government "Pirate"
		personality staying plunders disables
		system Mora
		fleet "Small Southern Pirates" 3
		fleet "Large Northern Pirates" 2

mission "Alexandria Library Card: Rescue: Escort"
	name `Escort Fleet: Deep Security`
	description `A Deep Security rescue fleet will escort you to Mora so you can repair the Gonza Book Cruise before <date>.`
	source Valhalla
	destination Ingot
	deadline 9
	to offer
		has "Alexandria Library Card: Return: done"
		not "Alexandria Library Card: Rescue: declined"
		not "Alexandria Library Card: Rescue: failed"
	on offer
		conversation

> 			`Deep Security meets you and Daemos. "A fleet is prepared, but only your ship can board the Gonza Book Cruise. We must bring it back to <destination> before <date>."`

Reviewer: ```suggestion
			`Deep Security meets you and Daemos. "A fleet is prepared, but only your ship can board the Gonza Book Cruise. We must be back before <date>."`
```

Author: The ship's destination should be specified. Also, the destination is not where you are at the time that statement is made. However, "back to" is incorrect as well.

Author: Fixed.

	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to archive the Chal Jub Paqmey."
	blocked `Daemos complains, "I can't sleep on a bookshelf. I need a bunk and space for 1 ton of cargo."`
	source Ingot
	destination Alexandria
	passengers 1
	landing
	clearance
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Rescue: done"
	on offer
		conversation
			`When you arrive at Ingot, you see some Seekers and Deep Security agents conversing with Daemos.`
			`	Daemos greets you with 60,000 credits and some kind words. "Just get these five books and I back to Alexandria and our Seek will be complete. You'll be a Journeyman Seeker with all the associated rights and privileges... and 100,000 more credits."`
			choice
				`	"Great! Let's leave."`
					accept
				`	"I'm done with your quest, old man. I don't want rights or privileges, and I certainly don't want 100,000 credits."`
					decline
	on visit
		dialog `You've forgotten your senile friend or his pencils. Perhaps they're in one of your escorts?`
	on complete
		set "library card"
		payment 100000
		set "license: Alexandria Seeker"
		conversation

> 			`When you land at the station, a gathering of librarians meet you at your ship. Daemos ceremoniously pins a gold icon of a book onto you and hands you a key card allowing access to the station and its library.`

Reviewer: Slight repetition of "you".
```suggestion
			`When you land at the station, a gathering of librarians meet you at your ship. Daemos ceremoniously pins a gold icon of a book to your shirt and hands you a key card allowing access to the station and its library.`
```

Author: I'd rather leave the ambiguity of where the pin was placed. Also, in this context, "onto you" can imply it is going onto something you are wearing.

			`	"There are rumors that the writer of these books is an ancient being of great power over many worlds. He may still be alive, and writing the later books. Nobody has seen or heard from him in centuries though. He may even be a myth."`
				goto questions
			label services
			`"It's a library, fool. What do you think we do? Serve bacon? Teach juggling lessons?`
			`	"We have books. Lots of books. If you want to read books, come here to our library."`
			label "library services"
			choice
				`	"Bacon?"`
					goto bacon
				`	"Juggling lessons?"`
					goto juggling
				`	"Books?"`
					goto "library books"
				`	"Change the subject."`
					goto questions
				`	"End the conversation."`
					goto end
			label bacon
			`"It was a joke. If you want bacon, go look in the Dirt Belt. There's plenty of food out there."`
				goto "library services"
			label juggling
			`"Are you daft? We don't provide juggling lessons here. Find such silliness on the Paradise Planets."`
				goto "library services"
			label "library books"
			`"Yes. We have the largest archive in all of Human space. If you're looking for a book you will find it here."`
				goto "library services"
			label jobs

> 			`"Yes.`

Author: This is inconsistent with Daemos's character. Also, this is a multi-line quote.

The later lines should not end with a " either, but I'll fix that in a separate commit.

> 			`"Yes.`

Author: Corrected.

			`	"The library will pay you to Seek for rare books or artwork, and bring them here for archival.`

> 			`	"We'll also pay you to ferry librarians or library materials between Alexandria Station and other libraries. Find those jobs in Job Boards here or elsewhere.`

Author: Corrected.

			`	"The library has a mandate to preserve forbidden knowledge. If you can reach pirate planets, you can find jobs to smuggle illegal literature to Alexandria.`
			`	"Once you've proven yourself, we may give you more difficult jobs: a book cruise, long-range escort missions, or hunting book thieves."`
				goto questions
			label end
			`"Excellent. Welcome to your new life as an Alexandria Seeker."`
	on decline
		fail "Access to Alexandria"
	on fail
		fail "Access to Alexandria"

mission "Access to Alexandria"
	name "Access to Alexandria"
	description "Access to Alexandria"

> 	repeat 2

Reviewer: Why `repeat`? (and even if it does have repeat, why exactly twice?)

Author: Sometimes the mission vanished once. This is my workaround. If it repeats endlessly, you end up with an infinite number of "Access to Alexandria" missions in your save file.

	source Earth
	destination Shroud
	passengers 1
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Offer: done"
	on offer
		conversation
			`When you arrive, you see Daemos Titan eager to depart.  He walks at a brisk pace towards your ship without even telling you his destination.  Porters behind push a cart with an impressive number of books and at least a thousand pencils.  You notice there is no modern equipment; not even a data pad.`
			`	"Let's go already," he says.  "The five Chal Jub Paqmey are not getting any younger."`
			`	"Go where?"`
			`	"Yes, yes, yes.  Details." He fumbles about his person and eventually finds a folded piece of paper in his beard.  He shoves it at you so fast that his hand almost hits you.  You're fairly certain it wasn't meant as a hostile gesture.  This man simply hasn't talked to anyone other than himself for so long that he's forgotten basic social graces.`
			choice
				`	Read the paper...`
			`The paper is of organic origin, not a modern synthetic, and is stamped with the Alexandria Library Seal using actual wax.  The text is badly smudged, but you can make out some details.`

> 			`	Your job is to ferry Daemos around the galaxy looking for the five Chal Jub Paqmey.  You'll receive a payment of 60,000 credits each time he finds one.  When you return Daemos and all five Chal Jub Paqmey to Alexandria Station in the Markeb system, you'll receive another 100,000 credits and an Alexandria Library Card.`

Author: This is not a quotation.  It is the main character's summary of what he/she/it can gather from the damaged letter.  Quoting the letter would be impossible since it is smudged to the point of being barely readable.

> 			`	Your job is to ferry Daemos around the galaxy looking for the five Chal Jub Paqmey.  You'll receive a payment of 60,000 credits each time he finds one.  When you return Daemos and all five Chal Jub Paqmey to Alexandria Station in the Markeb system, you'll receive another 100,000 credits and an Alexandria Library Card.`

Reviewer 3: Still, the section reads more like an omnipotent force is telling the player that this will happen, rather than the narrator telling the player what they can figure out.
```suggestion
			`	It appears your job is to ferry Daemos around the galaxy looking for the five Chal Jub Paqmey.  You'll receive a payment of 60,000 credits each time he finds one.  When you return Daemos and all five Chal Jub Paqmey to Alexandria Station in the Markeb system, you'll receive another 100,000 credits and an Alexandria Library Card.`
```

			`	The library card will allow permanent access to the Alexandria Station and its library, no matter your legal status, nor the political situation.  The library also offers Job Board jobs to card holders, and possibly a daily salary.`

> 			`	This sounds like a good deal.  Possibly too good.  Why are they offering this?  More to the point, why are they offering it to you?`

Author: This line is unnecessary.  Either the player already realizes the deal is too good to be true, or they'll stumble foolishly into a good deal.

					goto what
				`	"Let's go find the Chal Jub Paqmey."`
					goto accept
				`	"Go find another captain who will cater to your madness."`
					decline
			label what
			`	"They're a series of books that record an opera of a merchant-explorer named HoD Qav, wandering a world that seems much bigger than herself.  Or was it himself?  Anyway, nobody knows the ending since there were supposed to be eight books, but only five were made.  All five were lost over the centuries and now I'm going to find them."`
			`	"Where are they?"`
			`	"If I knew, I wouldn't be searching for them!  I just worked out where the first one is, but we'll have to scour space for the other four."`
			choice
				`	"Let's go find the Chal Jub Paqmey."`
					goto accept
				`	"Sorry, I don't have time for this."`
					decline
			label accept
			`	"Excellent.  Our first destination is planet Shroud in the Alheka system.  I suggest you take a route through the western Paradise Planets.  We should be pretty safe there."`
				accept
	on visit
		dialog `It seems you left the insane librarian or some of his thousand pencils behind.  Perhaps they're in one of your escorts?`
	on complete
		payment 60000
		conversation

> 			`Shroud is a planet of anachronisms.  In the high mountains, vast castles of medieval design house the pampered ultra-rich, living as if in a Victorian era.  Hidden behind expensive paintings can be found all the comforts of the modern era: virtual reality booths, food-synthesizing devices, massive panel screens for tracking stocks, and the like.  Down below, the poor live in cramped conditions no better than medieval serfs, using modern technology when required by their by employers.`

Author: That paragraph was intended to describe the journey of the player from the cloud cities down to the marsh, but it was written incorrectly.

			`	Daemos wasn't interested in any of that though.  He took you below the cloud cities of the rich, beyond the factory towns, past the countryside, through a marsh, into a hidden temple, beneath a fake support column, and down to a subterranean room.  In the center of the room, an impossibly old monk was meditating with his eyes closed.  He hovered a half meter above the ground humming in an almost guttural tone.  Within his hands was a book.`
			choice
				`	Approach the monk...`
			`When Daemos approached, the monk's eyes opened, and he uttered, "for the Seeker I do seek."  He handed Dameos the book and then vanished.`
			`	Daemos didn't look surprised at all; he calmly put the book in his satchel and handed you 60,000 credits.`
			`	"We have the Paq wa'DIch.  Next we seek the Paq cha'DIch.  Meet me in the spaceport when you're ready."`

> 			`	This was strangest mission you've had yet, and a pretty easy 60,000 credits.  You suspect the next four books will be harder to get.`

Author: My assumption was that the player would attempt this mission in the early game.  I do see how this dialog would look like a game bug if it was encountered in the middle of the wanderer story.  I might add a check on the date and adjust the dialog accordingly.

mission "Alexandria Library Card: Second Volume: Step 1"
	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."
	blocked `Daemos complains, "What do you think I am, an Anti-Missile Turret?  I need a bunk and space for 1 ton of cargo."`
	source Shroud
	destination Tinker
	passengers 1
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: First Volume: done"
	on offer
		conversation
			`	You meet Daemos at the spaceport with the book on his lap.  It's hard to see beneath his beard, but you think he's smiling.`
			`	"Are you ready to go, Captain <last> <first>?"`
			choice
				`	"I have some questions first."`
					goto questions
				`	"Let's go."`
					goto leave
			label questions
			`	"What do you want to know?"`
			choice
				`	"What was in the book?"`
					goto contents
				`	"How will you find the next book?"`
					goto seek
				`	"Who was that monk, and where did he go?"`
					goto monk
				`	"No more questions.  Let's go."`
					goto leave
				`	"I'm done with your quest.  Enjoy the book."`

> 					decline

Author: The option adds some color to the story.  The player has the freedom to ignore it.

	on complete
		conversation
			`You are lead to a vast skyscraper overlooking a mountain range.  From the glass-walled elevator, you take in the breathtaking view.  The elevator stops on the top floor and you see a well-dressed man surrounded by bookshelves.  He sitting on a gold-lined leather chair, pulled up to a marble desk.  The bookshelves also appear to be hand-carved marble.  Whoever this is, he is exceptionally rich.`
			choice
				`	Interrupt him...`
			`The man turns to you and says, "it is always an honor to meet Alexandria Seekers.  How can I help you?"`
			`	Daemos responds, "I seek the Paq cha'DIch, second volume of the Chal Jub Paqmey.  I've heard its your kind of literature.  Do you have it?"`
			`	"I don't have it now, but I can get it for you.  Do you have the book, 'A Thousand Poisons of Amazon and How to Conceal Their Taste?'  I've been looking for a signed first edition."`
			`	"Ekhh.  A trade.  Okay, I'll get you a copy."`
			`	On the way out, Daemos whispers to you, "we have to fly to planet Amazon in the Schedar system next.  I have the book, but not the signature.`
			`	"Meet me in the spaceport when you're ready to go."`

> 

Author: This is an "on complete" trigger, so "accept" has no purpose.  I'll be merging some missions to get rid of the "visit the spaceport" part.  Some sections will merge, and such changes will be necessary.

> 

Reviewer 3: Yeah, I just misread the syntax and didn't realise it was there.

mission "Alexandria Library Card: Second Volume: Step 2"
	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."
	blocked `Daemos complains, "I can't sleep in cooling ducts.  I need a bunk and space for 1 ton of cargo."`
	source Tinker
	destination Amazon
	passengers 1
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Second Volume: Step 1: done"
	on offer
		conversation
			`Daemos meets you at your ship and says, "our destination is in a dangerous area of Syndicate space.  I've arranged some escorts so you don't get me killed."`
			label questions
			choice
				`	"What will we do on Amazon?"`
					goto what
				`	"Why do we need escorts?"`
					goto why
				`	"Sounds good.  Let's go."`
					goto go
				`	"I don't have time to help you any more."`

> 					decline

Author: I'm keeping the "I don't want to help you" bits to add color to the story.

				goto questions
			label why
			`Daemos responds, "that area is crawling with pirates and I doubt your ship will survive alone.  The Chal Jub Paqmey are critical, so the Library is willing to expend significant resources to obtain it."`
			`	"Why am I your courier if you fear I'll get you killed?"`
			`	"Courier?  You're not a courier.  I don't understand."`
			`	"You're paying me to ferry you around the galaxy looking for books."`
			`	"No.  You don't understand.  You're my apprentice.  Alexandria Library doesn't ask just anyone to assist a Grandmaster Seeker.  They think you have potential to be a Seeker, despite all your faults.  The payments are just stipends to keep you fed during your apprenticeship."`
			label "seeker questions"
			choice
				`	"What is a Seeker?"`
					goto seeker
				`	"Why me?"`
					goto "why me"
				`	"What faults?"`
					goto faults
				`	Change the subject...`
					goto questions
				`	"Sounds good.  Let's go."`
					goto go

> 				`	"I don't have time to help you any more."`

Author: A fine suggestion.  I like it.

> 				`	"I don't have time to help you any more."`

Author: Actually, it doesn't work since the player won't know about the Seeker aspect unless they choose the right dialog options.  This makes me miss the `[show_if]` feature in WML.  There is no way have a conditionally-shown choice.

				goto "seeker questions"
			label faults
			`"You must know yourself if you are to be a successful Seeker.  You are never truly strong until you turn your very weaknesses into strengths."`
				goto "seeker questions"
			label go
			`"Excellent!  That's the spirit."`
				accept
	on decline
		fail "Alexandria Library Card: Second Volume: Step 2 Escort"
		fail "Alexandria Library Card: Second Volume: Step 2 Jump Instructions"
	on fail
		fail "Alexandria Library Card: Second Volume: Step 2 Escort"
		fail "Alexandria Library Card: Second Volume: Step 2 Jump Instructions"
	on visit
		dialog `It seems you left the insane librarian or some of his thousand pencils behind.  Perhaps they're in one of your escorts?`
	on complete
		conversation
			`Instead of landing on the main Amazon spaceport, Daemos has you land in a clearing deep inside one of the densest forests on the planet.  You climb through some aggressive vegetation, up a mountain, to a hut.  Daemos looks at you and says, "this is the place."`
			choice
				`	Open the door...`

> 			`When the door opens, Daemos rushes forward in a panic, before you can even survey the room.  It seems the man you came to see is dead, and quite recently too.`

Author: I'm going to rewrite this entire encounter.  It was poorly done.

> 			`When the door opens, Daemos rushes forward in a panic, before you can even survey the room.  It seems the man you came to see is dead, and quite recently too.`

Author: An investigation doesn't make sense.  Daemos and the player would be in a hurry to get out rather than getting implicated in the crime.  I've modified the dialog so it is left as a mystery.

> 			`	Daemos searches the room and finds the culprit.  The author was working on the second edition of his book, "A Thousand Poisons of Amazon and How to Conceal Their Taste."  Somehow, one of the poison recipes was swapped with the lunch recipe.  A poetic end to someone whose work has lead to many deaths in the Syndicate.`

Author: Yes.  I'll add another page of dialog on the examination.

			`	This seems to be the end of the quest, so you ferry Daemos back to the main Amazon spaceport.`
			choice
				`	Fly back to the main spaceport...`

> 			`"This will just delay us," says Daemos, "I know a group of businessmen on Deadman's Cove who can forge this signature."`

Author: The executive asked for a signed first-edition copy of a book.  Daemos had the book, but it was not signed.

> 			`"This will just delay us," says Daemos, "I know a group of businessmen on Deadman's Cove who can forge this signature."`

Reviewer 3: Wouldn't the correct word be "his" instead then?
```suggestion
			`"This will just delay us," says Daemos, "I know a group of businessmen on Deadman's Cove who can forge his signature."`
```

mission "Alexandria Library Card: Second Volume: Step 2 Escort"
	name "Escort Fleet: Deep Security"
	description "A fleet is paid to escort you to <destination>.  It will abandon you if you don't arrive by <date>."
	source Tinker
	destination Amazon
	deadline 15
	to offer
		has "Alexandria Library Card: Second Volume: Step 1: done"
		not "Alexandria Library Card: Second Volume: Step 2: declined"
		not "Alexandria Library Card: Second Volume: Step 2: failed"
	on offer
		conversation
			branch "can offer"
				has "Alexandria Library Card: Second Volume: Step 2: active"
			`A man from Deep Security approaches you.  "Where is Daemos?  We were hired to escort him?"`
			`"Sorry," you say, "but he's looking for another captain."`
				defer
			label "can offer"
			`A man from Deep Security approaches you.  "We've been asked to escort you two Seekers to <destination>.  If you don't arrive before <date>, then we'll have to depart for our duties elsewhere."`
				accept
	npc
		government "Deep Security"
		personality heroic escort disables
		fleet
			names "deep"
			variant
				"Corvette"
				"Corvette (Speedy)"

> 				"Headhunter" 3

Author: The escorts get their own dialog so that they mean more to the player than just cannon fodder.

		"Hyperdrive"

ship "Sparrow" "Sparrow (Hai Tracker Atomic)"
	outfits
		"Hai Tracker Pod" 1
		"Hai Tracker" 56
		"nGVF-BB Fuel Cell"
		"Hai Chasm Batteries" 1
		"Supercapacitor" 2
		"D14-RN Shield Generator"
		"A120 Atomic Thruster"
		"Chipmunk Plasma Steering"
		"Hyperdrive"

mission "Alexandria Library Card: Second Volume: Step 3"
	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."
	blocked `Daemos reminds you, "I won't fit in a torpedo.  I need a bunk and space for 1 ton of cargo."`
	source Amazon
	destination "Deadman's Cove"
	passengers 1
	clearance "Daemos sends a message to his friend on Deadman's Cove.  You receive coded landing coordinates."
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Second Volume: Step 2: done"
	on offer
		conversation
			`Daemos looks dead serious when you approach him.  He says, "we're going to a pirate planet.  That's an insanely dangerous thing to do in a ship like yours.  I've arranged for us to travel with a large group of bounty hunters.  You'll need to speak to the planet's authorities before they'll let you land."`

> 			`	This sounds like a quick but interesting death.  You ponder your options.`

Author: This line was redundant.  I took it out.

			label questions
			`"Do you want to know anything before we go?"`
			choice
				`	"How do I speak to the planet authorities?"`
					goto how
				`	"Who is escorting us?"`
					goto who
				`	"Sounds good.  Let's go."`
					goto go
				`	"I don't want to die.  Find another captain."`
					decline
			label how
			`(Game information)  To talk to a planet:`
			`	1. Select the planet.  Pressing the LAND button (default is "L") will select the first planet you can land on, which is Deadman's cove.`
			`	2. Hold down "shift" and press the TALK button (default is "T").  That will open a dialog to talk to the planet.`

> 			`	Daemos will then send a code, and you can land.`

Author: Actually, I already have an "on enter" dialog for this section, so it is entirely redundant anyway.

					goto sparrows
				`	"Will she escort us back?"`
					goto back
				`	Change the subject...`
					goto questions
				`	"Sounds good.  Let's go."`
					goto go
				`	"I don't want to die.  Find another captain."`
					decline
			label name
			`"That's exactly the kind of question you should not ask her if you want to stay alive."`
				goto "slayer questions"
			label sparrows
			`"With missiles and upgraded thrusters, a fleet of Sparrows can use highly-effective swarm tactics.  Besides, with that many escorts, the bounty hunters will have enough targets that they'll have trouble hitting us."`
				goto "slayer questions"
			label back
			`"Ah.... no.  Crimson Slayer has other places to be"`
			`	"So you're saying we won't need help getting back?"`
			`	"Uh... well...  Departing a planet is far more dangerous than landing.  Pirates will wait just outside of a planet and destroy you before you can even react.  A fast ship with good shields can dodge if it curves away from the planet as soon as it leaves orbit.  A ship with decent shields, good steering, and a Scram Drive can jump immediately.  Not your ship though.  Certainly not there."`

> 			`	You ask, almost sarcastically, "do you plan on us living on Deadman's Cove or just dying there?"`

Author: I changed this to a pair of choices that let the player discuss the necessity of longevity.

			`	"I'm sure something will come up.  Don't worry about it."`
				goto "slayer questions"
			label go
			`"That's the spirit, Captain <last> <first>!  As they say, 'Heghlu’meH QaQ jajvam!'"`
			`	"What does that mean?"`
			`	"The rough translation is, 'you worry too much.'"`
				accept
	on enter Almach

> 		dialog `Press "L" to land and then "shift-T" to talk to the planet.`

Author: That is too long for a dialog.  Those boxes are tiny.

> 		dialog `Press "L" to land and then "shift-T" to talk to the planet.`

Reviewer 3: The main reason I suggested this is more generally to suggest that the instructions about landing were in the dialog instead of the conversation

> 		dialog `Press "L" to land and then "shift-T" to talk to the planet.`

Author: The instructions need to be provided at the time that they must be followed.

			`	Daemos turns to you and says, "meet me at the spaceport when you are ready to return to Tinker.  Don't take long, and don't fall asleep in the hotels."`
	npc
		government "Pirate"
		personality vindictive plunders staying
		system Almach

		fleet
			names "pirate"
			variant
				"Vanguard"

mission "Alexandria Library Card: Second Volume: Step 3 Escort"
	name "Escort Fleet: Crimson Slayer"
	description "Some bounty hunters are paid to escort you to <destination>.  They will abandon you if you don't arrive by <date>."
	source Amazon
	destination "Deadman's Cove"
	deadline 4
	to offer
		has "Alexandria Library Card: Second Volume: Step 2: done"
		not "Alexandria Library Card: Second Volume: Step 3: declined"
		not "Alexandria Library Card: Second Volume: Step 3: failed"
	on offer
		conversation
			branch "can offer"
				has "Alexandria Library Card: Second Volume: Step 3: active"
			`Ahead, you see a woman wearing bloodstained leather clothes, holding a huge gun.  Her face is emotionless and her gaze is like the point of a sword.  "Where is Daemos?  We were hired to escort him?"`
			`	"He's looking for a different courier for his suicide mission."`
				defer

> 			label "can offer"

Author: I would prefer to introduce the characters that will escort you.  They should be more than just cannon fodder on the main screen.

			`Ahead, you see a woman wearing bloodstained leather clothes holding a huge gun.  Her face is emotionless and her gaze is like the point of a sword.  She says, "we're killing a bounty in <destination> soon and Daemos wants you to follow us.  If you don't arrive before <date>, then you're on your own."`
			`	You only have four days with these escorts, so you should not wander.`
				accept
	npc

> 		government "Merchant"

Author: Apparently there is an "Escort" government.

		personality heroic escort disables
		fleet
			names "bounty hunter"
			variant
				"Clipper (Speedy)"
				"Sparrow (Meteor Atomic)" 5
				"Sparrow (Javelin Atomic)" 5
				"Sparrow (Heavy Atomic)" 5

> 				"Sparrow (Hai Tracker Atomic)" 5

Author: The fleet is designed for effectiveness.  A Sparrow with 1000 top speed can be quite deadly with Hai trackers.  Sometimes a Korath Raider will show up in the star system and I had to make the Sparrow fleet durable enough to delay the raider for 15 seconds or so.

> 				"Sparrow (Hai Tracker Atomic)" 5

Reviewer 3: I can understand the balance aspect, but it just feels like every pirate/mercenary and their mother have copious amounts of Hai technology, despite Hai space being locked off to pirates and being difficult to find in the first place, unless you're trying to map all systems, which most pirates definitely do not have the resources to do.

> 				"Sparrow (Hai Tracker Atomic)" 5

Author: I'm rearranging the contents of the fleet.

				"Sparrow" 12

mission "Alexandria Library Card: Second Volume: Step 4"
	name "Alexandria Library Card"
	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."
	blocked `Daemos complains, "Where do you expect to put me?  A gun holster?  I need a bunk and space for 1 ton of cargo."`
	source "Deadman's Cove"
	destination Tinker
	passengers 1
	cargo "old books and a thousand pencils" 1
	to offer
		has "Alexandria Library Card: Second Volume: Step 3: done"
	on offer
		conversation
			`Several extremely large men in masks meet you and Daemos at the spaceport.`
			`	Daemos informs you, "these legitimate businessmen will accompany us back to Tinker."`
			`	You begin to ask, "what-"`
			`	"No discussions," warns Daemos.  "Let's leave."`
			`	For a moment you think there may be a hint of green skin under one of the masks.  It's probably just your fear running wild.  Regardless, you'd rather die on a different planet, and a long time from now.  It is definitely time to leave.`

> 				accept

Author: I want a new player to have the opportunity to check out the pirate job board.  The fact that they are being pressured to leave doesn't mean that they'll take the advice.

			`	"Meet me at the spaceport when you're ready."`
	npc
		government "Pirate"
		personality vindictive plunders staying
		system Diphda
		fleet
			names "pirate"
			variant
				"Osprey"
				"Manta"
				"Fury (Bomber)"
				"Fury (Heavy)"
				"Fury (Missile)"

mission "Alexandria Library Card: Second Volume: Step 4 Escort"
	name `Escort Fleet: "Legitimate Businessmen"`
	description `"Legitimate businessmen" agree to escort you to <destination>.  They say if you don't arrive by <date>, then "your fate is in your own hands."`
	source "Deadman's Cove"
	destination Tinker
	deadline 16
	to offer
		has "Alexandria Library Card: Second Volume: Step 3: done"
		not "Alexandria Library Card: Second Volume: Step 4: declined"
		not "Alexandria Library Card: Second Volume: Step 4: failed"
	on offer
		conversation
			`The largest of the men speaks.  "We'll follow you to <destination>, as agreed.  If you don't arrive by <date>, your fate is in your own hands.  I suggest you leave now."`
				accept
	npc

> 		government "Merchant"

Author: I could go with the "Marauder Escort" government and make them orange or red.  The bounty hunter escorts could be handled similarly.  I'll try it out and see what looks best on the screen.

				"Marauder Leviathan (Engines)" 2
				"Marauder Firebird (Weapons)" 3
				"Marauder Raven (Weapons)" 3
	npc
		government "Quarg"
		personality vindictive staying
		system Sadalmelik
		fleet
			names "quarg"
			variant
				"Quarg Skylark"
				"Quarg Wardragon"

mission "Alexandria Library Card: Third Volume: Step 2 Escort"
	name `Escort Fleet: Navy Patrol`
	description `A Republic Navy patrol group will escort you to <destination>.  After <date>, they'll have to return to their patrol.`
	source Rand
	destination Grakhord
	deadline 16
	to offer
		has "Alexandria Library Card: Third Volume: Step 1: done"
		not "Alexandria Library Card: Third Volume: Step 2: declined"
		not "Alexandria Library Card: Third Volume: Step 2: failed"
	on offer
		conversation
			`"Greetings, Daemos Titan and Captain <first> <last>!  I'm Admiral Tamarak, commander of the Navy Seventh Patrol Group.  I've never been asked to escort a Grandmaster Seeker before.  It's quite an honor.`
			`	"I'll take you to <destination> as long as you let me go by <date>.  The patrol route can't be left alone for long."`
				accept
	npc
		government "Republic"
		personality heroic escort disables
		fleet

> 			names "pirate"

Author: Oopes.  An entertaining mistake though, now that I noticed it.

> 			names "pirate"

Author: "This is my Navy carrier escort, 'Medusa's Impaler' and my frigate escort, 'God.'"

			`	After the lecture, the woman, Dr. Sophia Nance, runs over and hugs Daemos.  "Dr. Demon!  I've missed you."`
			`	"I haven't seen you since your defense," he says to her.  "What way is that to treat your thesis advisor?"`
			`	"Sorry, Dr. Demon.  The Quarg pulled me into this project, and I was too excited to refuse.  I've been out of Human space in a Quarg Wardragon for a while."`
			choice
				`	Continue...`
			`She pulls an ancient book out of her backpack, and hands it to Daemos.  "Will this make up for my absence?  We found it on an uninhabited world to the north-east of here."`
			`	Daemos exclaims, "the Paq LoSDIch!  You are an angel, Sophia.  What about the symbols from Rand?"`
			`	"Bad news there, I'm afraid.  Neither I nor the Quarg could translate them.  We suspect they're not linguistic, but rather a map of sorts.  The Quarg rotated them, and some looked almost like constellations.  They say it may be a map, but if it is, it's for a younger era in this galaxy.  Stars shift around pretty far after a few hundred millennia."`
			choice
				`	Look at the map...`
			`Sophia hands the map to Daemos.  You look over his shoulder.`

> 			`	"I see it," you say.  "The dots and lines are the star locations.  The crosses and ovals show which planet or moon by the orbit.  Everything is slightly out of place, but I can recognize the constellations.  I know which planets we need to visit."`

Author: The player is a pilot.  It is expected that he would understand basic navigation and constellations.  I will see how I can rearrange this text.

			`	"That doesn't tell us where the keys are on the planets though," warns Daemos.`
			`	Sophia smiles, "on that matter, I can help you.  My Quarg collaborators say the keys should appear on the Navy's gamma ray sensors as glitches, filtered out by quality control code.  They already have the locations in their sensor logs' raw data stream.  You just have to tell them the frequencies to look for."`

> 			`	She hands you a data pad and says, "take this to the naval base on New Austria.  They'll know what to do."`

Author: The planet description says they are building one, all the way from the beginning of the game.  You can't build vast hangers in a few days.  It takes months or years, especially if you have to wait for a galactic government to allocate funds.  It is reasonable for the surveillance aspects to be set up before the hangers.  Also, you are informed that the base is secret.

> 			`	She hands you a data pad and says, "take this to the naval base on New Austria.  They'll know what to do."`

Author: I'm changing it to a Navy research center.

			choice
				`	Continue...`
			`You meet Daemos afterwards, and he's in a hurry to continue.`

> 			`	"Another 60,000 credits, as promised.  Are you ready to see a secret Navy base?"`

Author: The man is senile.  "Secret" could just mean he forgot about it.  Alzheimer's disease has that effect on someone.

	npc
		government "Pirate"
		personality vindictive plunders staying
		system Menkent
		fleet
			names "pirate"
			variant
				"Marauder Leviathan (Engines)" 1
				"Aerie" 2
				"Dagger" 4

mission "Alexandria Library Card: Third Volume: Step 3 Escort"
	name `Escort Fleet: Tarazed Shipyards`
	description `Tarazed Shipyards corporate ships will escort you to <destination>.  After <date>, they'll have to return to their patrol.`
	source Grakhord
	destination "New Austria"
	deadline 19
	to offer
		has "Alexandria Library Card: Third Volume: Step 2: done"
		not "Alexandria Library Card: Third Volume: Step 3: declined"
		not "Alexandria Library Card: Third Volume: Step 3: failed"
	on offer
		conversation
			`A saleswoman in a Tarazed Shipyards uniform approaches you.  "We have business near the Navy base so we'll escort you to <destination>.  If you don't arrive by <date>, we'll have to part ways.`
			`	"By the way, I noticed you may be in need of a new ship.  You'll pass by the Tarazed system when you exit Quarg space.  Our corporate capital there, on Wayfarer, has the most well-stocked shipyard and outfitters in Human space.  Stop by if you have a chance."`
				accept
	npc

> 		government "Merchant"

Author: Tarazed doesn't go green until well into the Free Worlds story.  Merchants are always to be expected there though, especially if they are saleswomen.

> 		government "Merchant"

Reviewer 3: Militia (not FW) always have a consistent presence throughout the Dirt Belt and lower, even outside of FW space.

> 		government "Merchant"

Author: Only if you try to Dominate their planets.

> 		government "Merchant"

Reviewer 3: Militia show up in fleets, not just tributes. Tarazed has no direct militia spawns, but Dabih does.

> 		government "Merchant"

Author: I was going to go with "Merchant" for storyline reasons, but I went with "Independent" so they would be differently-colored than everything else.

				goto questions
			label accept
			`"Finally."`
				accept
	on visit
		dialog `You've forgotten your senile friend or his pencils.  Perhaps they're in one of your escorts?`
	on complete
		conversation
			`A Deep Security officer runs towards you and Daemos in a hurry yelling, "it's been disabled!  We have to leave!"`
			`	When the officer reaches you, he explains the Alexandria Seeker who was supposed to meet you is in grave trouble.  The Seeker's ship is disabled in an uninhabited system called Mora.'  The Seeker, Gonza Doorin, refuses to let anyone onto his ship except another Seeker.  There is a pirate fleet in the system, attacking anyone who gets near the ship."`
			`	Daemos interjects, "can Deep Security escort us?"`
			`	"Of course," the officer responds.  "We must leave at once."`
			`	"You'll need to refuel at Tania Australis before heading to Mora," warns Daemos.  "Meet us in the spaceport as soon as you're ready to go."`

> government "Librarian"

Author: That would make the mission impossible.  The Star Queen would be destroyed in seconds.

Also, the library maintains peaceful relations with the pirates.  One of the Seeks I'm planning involves setting up a distribution network of illegal literature printed in anarchist planets with pirates paid to smuggle the books into civilized space.  Surely as an anarchist yourself, you would understand the appeal.

In the Alexandria Library Card mission, Daemos Titan is expected by the marauders and greeted as a business client rather than their next victim.  He says he "knows someone" that can forge a signature, implying he has done business with the marauders before.

			`	"More struggles across the world begin fifth book as the chaos escalates and expands.  HoD Qav helps resolve disputes, defend the weak, and steal from the rich.  The world descends into chaos with only one merchant holding back the inevitable."`
			choice
				`	"What happens next?"`
				`	Change the subject.`
					goto questions
			`"Next?  How can I know what happens next?  Only five of the eight books were ever written."`
			`	"There are rumors that the writer of these books is an ancient being of great power over many worlds.  He may still be alive, and writing the later books.  Nobody has seen or heard from him in centuries though.  He may even be a myth."`
				goto questions
			label services
			`"It's a library, fool.  What do you think we do?  Serve bacon?  Teach juggling lessons?`
			`	"We have books.  Lots of books.  If you want to read books, come here to our library."`
			label "library services"
			choice
				`	"Bacon?"`
					goto bacon
				`	"Juggling lessons?"`
					goto juggling
				`	"Books?"`
					goto "library books"

> 				`	"Change the subject."`

Reviewer: Why was this marked as resolved? The quotes are still there. (likewise for next two comments)
